.gitattributes

```diff
@@ -0,0 +1,4 @@
+# Prevent the default "Windows-style checkouts" that leads to mysterious
+# failures when cloning with default Git settings on Windows, e.g.
+# https://github.com/chevah/pythia/issues/3.
+* text=auto
```

Comment:
This might work for pythia ... but in general I am -1 for this.

I rather prefer to have Git configure to just leave the content of the file as it is without any convertion.

Do you know which files were triggering the errors ?

Maybe instead of `*` use a selector only for those files.

I think that is better to update the workflow so that git is reconfigured before a checkout 

This is what we ask from our developers https://trac.chevah.com/wiki/Development/Environment#configure-git

and this is how we should configure the slaves or github actions workflows.

Reply:
You are right in principle, this is why the next commit has fixed this. It's called _Fixed .gitattributes. Regenerated sha512.sum file for Windows._ The updated `.gitattributes` file:
```
# Prevent the default "Windows-style checkouts" that leads to mysterious
# failures when cloning with default Git settings on Windows, e.g.
# https://github.com/chevah/pythia/issues/3. More on this at
# https://github.com/actions/checkout/issues/135#issuecomment-613329879.
* -text
```

Is this fine with you? It's actually just an alignment with our common `.gitconfig` option:
```
[core]
    autocrlf = false
```

No need to also set this in workflows, as this is repo-wide. Should be default for all our repos to avoid future similar issues.

Reply:
You are right... Is better to have this attribute in all repos .. and also add .gitconfig.

I was not aware of  the worktree config file ... but instead of worktree .gitconfig maybe we can  use pre-repo .git/config config

```
Config file location
    --global              use global config file   ->    GIT internal -- I think
    --system              use system config file   -> User config  ~/.gitconfig
    --local               use repository config file   -> Repo  .git/config
    --worktree            use per-worktree config file  -> .gitconfig
```

So yes, if we can push these changes to the repo, it should be great...and we should push them to any other repo from chevah